projects/media-viewer/src/lib/toolbar/toolbar-event.service.spec.ts

```diff
@@ -29,13 +29,25 @@ describe('Toolbar Event Service', () => {
     service.toggleHighlightMode();
     expect(service.highlightModeSubject.getValue()).toBeFalsy();
   });
-  it('should turn off highlightMode is drawModel is toggled on', () => {
+  it('should turn off highlightMode if drawModel is toggled on', () => {
     service.highlightModeSubject.next(true);
     service.drawModeSubject.next(false);
     service.toggleDrawMode();
     expect(service.highlightModeSubject.getValue()).toBeFalsy();
   });
 
+  it('should set showCommentSummary to true if comment summary is toggled off', () => {
+    service.showCommentSummary.next(false);
+    service.displayCommentSummary();
```

Comment:
should this be called something like toggle comment summary, since it doesn't always display it